packages/plugins/puppetlabs-stdlib/puppetlabs-stdlib.spec

```
@@ -0,0 +1,76 @@
+%global commit 45454b8de4ff8f860f6f78438107133e510336ed
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:		puppetlabs-stdlib
+Version:	4.25.1
+Release:	2%{?dist}
+Summary:	Puppet Labs Standard Library
+License:	ASL 2.0
+URL:		https://github.com/puppetlabs/puppetlabs-stdlib
+Source0:	https://github.com/puppetlabs/puppetlabs-stdlib/archive/%{commit}/puppetlabs-stdlib-%{shortcommit}.tar.gz
```

How about using the one from the forge, which is the official build:
```suggestion
Source0:	https://forge.puppet.com/v3/files/%{name}-%{version}.tar.gz
```

This also means `%prep` needs to be modified to be more standard.